backend/server/init.go

```diff
@@ -152,6 +153,20 @@ func (s *Server) getInitSetting(ctx context.Context) (string, time.Duration, err
 		return "", 0, err
 	}
 
+	maximumSQLResultSizeSetting, err := protojson.Marshal(&storepb.MaximumSQLResultSizeSetting{
```

Comment:
I'm trying now to write the default size into store just in case we want to change the default later.

Reply:
Then I consider not to initial this setting, we will fallback to the default value if the setting not exists

Reply:
SGTM, either initialize the setting with 0, or don't initialize this setting. Caching is the only thing to look at, just to make sure we don't query the setting on every API read and creating driver from factory.